Snowflake.Data.Tests/Mock/MockCloseSessionGone.cs

```diff
@@ -29,6 +29,11 @@ public Task<HttpResponseMessage> GetAsync(IRestRequest request, CancellationToke
             return Task.FromResult<HttpResponseMessage>(null);
         }
 
+        public HttpResponseMessage Get(IRestRequest request)
+        {
+            return null;
+        }
+
         public T Post<T>(IRestRequest postRequest)
         {
             return Task.Run(async () => await PostAsync<T>(postRequest, CancellationToken.None)).Result;
```

Comment:
So in theory this could cause deadlock as well right?

Reply:
Yes